_src/field-attrs.md

```diff
@@ -38,6 +38,10 @@
   with arbitrary string keys. The [struct flattening](attr-flatten.md) page
   provides some examples.
 
+  *Note:* This attribute clashes with [`flatten`](attr-flatten.md) and might
```

Comment:
If I am reading this correctly, this line says that `flatten` clashes with `flatten`. Clashing attributes is bad, but it's not quite that bad!